src/Modules/ModParser.lua

```diff
@@ -759,6 +764,8 @@ local modNameList = {
 	["bleed duration"] = { "EnemyBleedDuration" },
 	["bleeding duration"] = { "EnemyBleedDuration" },
 	["bleed duration on you"] = "SelfBleedDuration",
+	["to blind enemies on hit"] = { "BlindChance" },
+	["Supported skills have 100% chance to blind enemies on hit"] = { "BlindChance" },
```

Comment:
Guessing you put this here for the blind support gem, handling for that is done in the skill gem which we already handle